examples/wordpress/README.md

```diff
@@ -0,0 +1,153 @@
+# How To Use NFS Persistent Volumes
+
+The purpose of this guide is to help you understand storage provisioning with NFS by creating a WordPress blog and MySQL database.
+Both the blog and database require persistent storage.  
+
+This guide assumes knowledge of OpenShift fundamentals and that you have a cluster up and running.
```

Comment:
You might want to reference same-app here like https://github.com/openshift/origin/tree/master/examples/jenkins does.